pkg/new-ui/v1beta1/backend.go

```diff
@@ -104,6 +129,53 @@ func (k *KatibUIHandler) CreateExperiment(w http.ResponseWriter, r *http.Request
 	}
 }
 
+func (k *KatibUIHandler) FetchExperiments(w http.ResponseWriter, r *http.Request) {
+
+	// check if user's username is provided in request Header
+	user, err := GetUsername(r)
+	if err != nil {
+		log.Printf("No user provided in kubeflow-userid header.")
+		http.Error(w, err.Error(), http.StatusUnauthorized)
+		return
+	}
+
+	namespaces, ok := r.URL.Query()["namespace"]
+	if !ok {
+		log.Printf("No 'namespace' query parameter was provided.")
+		err := errors.New("no 'namespace' query parameter was provided")
+		http.Error(w, err.Error(), http.StatusBadRequest)
+		return
+	}
+
+	namespace := namespaces[0]
+
+	err = IsAuthorized(user, "list", namespace, "experiments", "", "", experimentv1beta1.SchemeGroupVersion, k.sarClient)
```

Comment:
Also, please if K8s APIs doesn't have constants for CRUD actions (e.g. `list`). Let's add them to our [constants](https://github.com/kubeflow/katib/blob/master/pkg/controller.v1beta1/consts/const.go).

Reply:
Done!